### ISSUE TITLE
[none] Using proxy when installing chart

### ISSUE THREAD

**Issue by barkahmartiyana** [prometheus-community/helm-charts#3980]
### Is your feature request related to a problem ?

can we deploy it with proxy server , because we want install it in AWS outpost and the cluster can't connect to internet , we must connect proxy first to connect internet 

### Describe the solution you'd like.

deploy with proxy 

### Describe alternatives you've considered.

deploy with proxy

**Comment by zeritti** [comment]
> can we deploy it with proxy server , because we want install it in AWS outpost and the cluster can't connect to internet , we must connect proxy first to connect internet

Yes, helm understand the corresponding environment variables when fetching chart resources, see [net/http](https://pkg.go.dev/net/http#ProxyFromEnvironment).
